### PR TITLE
[Users] [FE] Create User detail page

### PR DESCRIPTION
**Pull request by bataimx**
Run "npm install" before do smoke test for this MR

### CODE REVIEW

File: src/pages/User/TasksSession.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  const { tickets: [tickets, setTickets] } = useContext(KPIStoreContext);
+  const assignTickets = tickets.filter((item) => item.user_id === userid && item.sprint_id === sprintID);
+
+  const fetchTasks = () => {
+    GetTasks(userid, sprintID).then((results) => {
+      const keyList = results.data.sprint_tickets.map((item) => item.key);
+      const ticketsMap = results.data.sprint_tickets.map((item) => {
+        // TODO BE return sprint_id. since 1 ticket might have more than 1 sprint_id
+        // item.sprint_id = '1,2,3'
+        item.sprint_id = sprintID;
```
Comment (bataimx) [inline src/pages/User/TasksSession.js]: @TShida951 we need BE return sprint_id here,
Cause 1 ticket might belong to multiple Sprint

Comment (bataimx) [reply src/pages/User/TasksSession.js]: Screen Shot 2019-12-02 at 17 07 17

**Comment by bataimx** [comment]
@TShida951 please review this PR when you have time, Thankyou

**Comment by TShida951** [comment]
Show 7 days or change label to `Last 5 days` please

スクリーンショット 2019-12-03 17 13 53

**Comment by bataimx** [comment]
@TShida951 i've just fixed  https://github.com/TShida951/kpi_management_front/pull/2#issuecomment-561098095

**Pull request merged**